### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by phil1995**
The following dependencies have been updated:
- cloud-access-swift (from `1.3.0` to `1.4.0`)
- GRDB (from `4.14.0` to `5.25.0`)

For the major update from GRDB 4 to GRDB 5, only changes regarding the ValueObservations were necessary. Because they now have a different runtime behavior (for more details see the [official GRDB 5 Migration Guide](https://github.com/groue/GRDB.swift/blob/e02f2c8abacff2799ed14926edcbf6e76fb9f805/Documentation/GRDB5MigrationGuide.md#runtime-behavior-of-valueobservation)).

### CODE REVIEW

File: Cryptomator.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -19,6 +19,15 @@
...
       },
+      {
+        "package": "AWSiOSSDKV2",
+        "repositoryURL": "https://github.com/aws-amplify/aws-sdk-ios-spm",
+        "state": {
+          "branch": null,
+          "revision": "443e35abb0646fb685e0754db74249d3242625ab",
+          "version": "2.27.10"
+        }
+      },
```
Comment (tobihagemann) [inline Cryptomator.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:30]: I suppose this should be added in a different PR, right?

**Review by tobihagemann — CHANGES_REQUESTED** [review]

File: Cryptomator.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved (hunk 1 above)
Comment (phil1995) [reply Cryptomator.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:30]: This comes automatically through the update of cloud-access-swift, which already has the dependency to `AWSiOSSDKV2`. Therefore, as already discussed internally, this can be considered resolved.

**Review by tobihagemann — APPROVED** [review]
